ipmininet/router/config/bgp.py

```diff
@@ -507,7 +512,7 @@ def __init__(self, base: 'Router', node: str, v6=False):
 
     @staticmethod
     def _find_peer_address(base: 'Router', peer: str, v6=False) \
-            -> Tuple[Optional[str], Optional['Router']]:
+            -> Tuple[Optional[str], Optional['Router'], Optional[str]]:
         """Return the IP address that base should try to contact to establish
         a peering"""
```

Comment:
I think that it would be better to update the documentation of this method

Reply:
Changed: 4af2eb9c67192d505d51b10d1ca6f22936272221